inbc-program/inbc/parser/source_app_parser.py

```diff
@@ -13,7 +13,7 @@
 
 
 def application_add(args: argparse.Namespace) -> str:
-    if (args.gpgKeyUri and args.gpgKeyName is None) or (args.gpgKeyName and args.gpgKeyUri is None):
+    if bool(args.gpgKeyUri) != bool(args.gpgKeyName):
```

Comment:
Consider using a more explicit conditional check for `args.gpgKeyUri` and `args.gpgKeyName` to improve readability. [medium]

Reply:
Ignoring, since this was the recommendation from the previous PR by the same tool.